packages/web-twig/src/Resources/components/Header/README.md

```diff
@@ -46,126 +106,394 @@ With Html syntax lexer (enabled by default):
 
-#### Available props
+👉 As of now, only single-level navigation is supported. You may consider
+using the [Header Dialog](#header-dialog) for other use cases such as the user
+menu.
 
-| Name           | Type   | Description                                |
-| -------------- | ------ | ------------------------------------------ |
-| `label`        | string | Label of the close button                  |
-| `ariaControls` | string | Target element which is controled by close |
-| `class`        | string | Additional class name                      |
+```twig
+<HeaderNav>
+  <HeaderNavItem>
+    <HeaderLink href="/" isCurrent>Job offers</HeaderLink>
+  </HeaderNavItem>
+  <HeaderNavItem>
+    <HeaderLink href="/">Part-time jobs</HeaderLink>
+  </HeaderNavItem>
+  <HeaderNavItem>
+    <HeaderLink href="/">Inspiration</HeaderLink>
+  </HeaderNavItem>
+  <HeaderNavItem>
+    <HeaderLink href="/">Replies</HeaderLink>
+  </HeaderNavItem>
+  <HeaderNavItem>
+    <HeaderLink href="/">Employers</HeaderLink>
+  </HeaderNavItem>
+</HeaderNav>
+```
 
-### NavbarToggle
+Both links and buttons are supported:
 
-#### Available props
+```twig
+<HeaderNav>
+  <HeaderNavItem>
+    <HeaderLink href="/">Link item</HeaderLink>
+  </HeaderNavItem>
+  <HeaderNavItem>
+    <HeaderButton onClick="console.log('Hello!')">Button item</HeaderButton>
+  </HeaderNavItem>
+</HeaderNav>
+```
 
-| Name           | Type                               | Description                                |
-| -------------- | ---------------------------------- | ------------------------------------------ |
-| `label`        | string                             | Label of the toggle button                 |
-| `ariaControls` | string                             | Target element which is controled by close |
-| `color`        | `primary`, `secondary`, `inverted` | Colors                                     |
-| `class`        | string                             | Additional class name                      |
+##### HeaderNav API
 
-### Nav
+| Prop name | Type     | Default | Required | Description      |
+| --------- | -------- | ------- | -------- | ---------------- |
+| `class`   | `string` | `null`  | no       | Custom CSS class |
 
-#### Available props
+##### HeaderNavItem API
 
-| Name    | Type   | Description           |
-| ------- | ------ | --------------------- |
-| `class` | string | Additional class name |
+| Prop name | Type     | Default | Required | Description      |
+| --------- | -------- | ------- | -------- | ---------------- |
+| `class`   | `string` | `null`  | no       | Custom CSS class |
 
-### NavItem
+##### HeaderLink API
 
-#### Available props
+| Prop name   | Type      | Default | Required | Description               |
+| ----------- | --------- | ------- | -------- | ------------------------- |
+| `class`     | `string`  | `null`  | no       | Custom CSS class          |
+| `href`      | `string`  | `#`     | no       | Link URL                  |
```

Comment:
Why isn't `href` required?

Reply:
I made it "required". But having done that, I found out it has technically _no effect_—no 500 or anything… Also on [our list](https://www.notion.so/almacareer/Twig-Povinn-props-je-mo-n-nevyplnit-aa2c55517687461dbfc9079248b15464?pvs=4).